R/utilities-sensitivity-analysis.R

```diff
@@ -623,6 +635,29 @@ plotTornado <- function(data,
   return(tornadoPlot)
 }
 
+
+#' @title lookupPKParameterDisplayName
+#' @param output an Output object
+#' @param pkParameter a string with the name of the pkParameter from the population sensitivity results index file
+#' @return the display name of the input pk parameter or the string pkParameter itself if no display name found
+lookupPKParameterDisplayName <- function(output,pkParameter){
+  pkParameterDisplayName <- NULL
+
+  for (pk in output$pkParameters){
+    # print("-----")
+    # print(paste("pkParameter is",pkParameter))
+    # print(paste("pk object is"))
+    # print(pk)
+    # print("^^^^^")
+    if ( pkParameter %in% pk$pkParameter ) {
+      pkParameterDisplayName <- pk$displayName

```

Comment:
If found, return directly. No need to break

Reply:
Fixed